firmware/usb.c

```diff
@@ -390,7 +390,7 @@ void usb_cb_completion(void) {
 			usb_ep_handled(USB_EP_FLASH_IN);
 		}
 	} else if (altsetting == ALTSETTING_PIPE) {
-		if (usb_ep_pending(USB_EP_PIPE_OUT)) {
+		if (booted && usb_ep_pending(USB_EP_PIPE_OUT)) {
 			pipe_usb_out_completion();
 			usb_ep_handled(USB_EP_PIPE_OUT);
```

Comment:
You'll keep receiving the interrupt over and over again until you call `usb_ep_handled` to clear the interrupt. If you want to prevent a USB transfer from taking place, don't call `usb_ep_start_out` until you're ready to receive data. Then the hardware will NAK the packet.